recipes/itk/all/patches/0001-find-conan-packages.patch

```diff
@@ -140,7 +140,7 @@
 -itk_module_target(itkopenjpeg NO_INSTALL)
 +find_package(OpenJPEG REQUIRED)
 +add_library(itkopenjpeg INTERFACE)
-+set_property(TARGET itkopenjpeg PROPERTY INTERFACE_LINK_LIBRARIES OpenJPEG::OpenJPEG)
++set_property(TARGET itkopenjpeg PROPERTY INTERFACE_LINK_LIBRARIES openjp2)
```

Comment:
should we fix target name in openjpeg?

Reply:
Are they incorrect? If yes, I agree changing

Reply:
`openjp2` is the official target